src/test/java/testPages/HomePageTest.java

```diff
@@ -0,0 +1,23 @@
+package testPages;
+
+import base.TestBase;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import steps.HomePageTestSteps;
+
+public class HomePageTest extends TestBase {
+    HomePageTestSteps homePageSteps;
+
+    @BeforeClass
+    public void setUp() {
+        initialization();
+        homePageSteps = new HomePageTestSteps();
+    }
+
+    @Test(priority = 1, description = "Check a selection of related products for the 'mobile phone' section")
```

Comment:
test description doesn't match the test, please update either test description or test